packages/perseus/src/widgets/__tests__/radio/choice-icon_test.jsx

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import {render, screen} from "@testing-library/react";
+import * as React from "react";
+
+import "@testing-library/jest-dom"; // Imports custom mathers
+
+import ChoiceIcon from "../../radio/choice-icon.jsx";
+
+const SAT_ICON_SIZE = 25;
+const LIBRARY_ICON_SIZE = 24;
```

Comment:
**nit:** These are defined in another file.  Can you import them from that other file instead of redefining them?